### ISSUE TITLE
[BUG] Unable to run NAS using the commands from the readme file

### ISSUE THREAD

**Issue by bigwater** [deephyper/deephyper#51]
**Describe the bug**
I followed the instructions of Quickstart section in the README.md. However, an error was generated when I run the following command. 

`deephyper nas ambs --evaluator ray --problem deephyper.benchmark.nas.polynome2Reg.Problem --n-jobs 1`

The error text looks like there is no `dense_skipco` module in this repository. 

```
  File "/home/hyliu/anl/work/deephyper/deephyper/benchmark/nas/polynome2Reg/__init__.py", line 1, in <module>
    from deephyper.benchmark.nas.polynome2Reg.problem import Problem
  File "/home/hyliu/anl/work/deephyper/deephyper/benchmark/nas/polynome2Reg/problem.py", line 3, in <module>
    from deephyper.nas.space.dense_skipco import create_search_space
ModuleNotFoundError: No module named 'deephyper.nas.space.dense_skipco'
```

I tried another command but ended up in another error. 
```
deephyper nas ambs --evaluator subprocess --problem deephyper.benchmark.nas.linearReg.Problem
```

and the error message: 
```
(test001) [hyliu@in02 deephyper]$ deephyper nas ambs --evaluator subprocess --problem deephyper.benchmark.nas.linearReg.Problem
2021-01-26 23:20:07.103393: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
 ************************************************************************
   Maximizing the return value of function: deephyper.nas.run.alpha.run
 ************************************************************************
Uncaught exception <class 'TypeError'>: __init__() got multiple values for argument 'num_workers'Traceback (most recent call last):
  File "/home/hyliu/miniconda/miniconda3/envs/test001/bin/deephyper", line 33, in <module>
    sys.exit(load_entry_point('deephyper', 'console_scripts', 'deephyper')())
  File "/home/hyliu/anl/work/deephyper/deephyper/core/cli/cli.py", line 42, in main
    args.func(**vars(args))
  File "/home/hyliu/anl/work/deephyper/deephyper/core/cli/nas.py", line 37, in main
    search_obj = search_cls(**kwargs)
  File "/home/hyliu/anl/work/deephyper/deephyper/search/nas/ambs.py", line 69, in __init__
    **kwargs,
TypeError: __init__() got multiple values for argument 'num_workers'
```

**To Reproduce**
To set up the environment, I made these steps. 
1. `conda create --name test001 python=3.7`
2. `conda activate test001`
3. `mkdir work && cd work`
3.  `git clone git@github.com:deephyper/deepspace.git`
4.  `git clone git@github.com:deephyper/deephyper.git`
5. `pip install -e deephyper`
6. `pip install -e deepspace`

Then, run the two commands to generate the error messages above: 

```
deephyper nas ambs --evaluator ray --problem deephyper.benchmark.nas.polynome2Reg.Problem --n-jobs 1
```

```
deephyper nas ambs --evaluator subprocess --problem deephyper.benchmark.nas.linearReg.Problem
```

**Expected behavior**
The framework does the NAS on the two problems. 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Red Hat Enterprise Linux Server release 7.2 
 - Python version 3.7.9
 - DeepHyper Version a267505a

`pip list`
```
Package                  Version             Location
------------------------ ------------------- ------------------------------------
absl-py                  0.11.0
aiohttp                  3.7.3
aiohttp-cors             0.7.0
aioredis                 1.3.1
antlr4-python3-runtime   4.7.2
astunparse               1.6.3
async-timeout            3.0.1
attrs                    19.1.0
blessings                1.7
cachetools               4.2.1
certifi                  2020.12.5
chardet                  3.0.4
click                    7.1.2
cloudpickle              1.6.0
colorama                 0.4.4
colorful                 0.5.4
ConfigSpace              0.4.16
cycler                   0.10.0
Cython                   0.29.21
deap                     1.3.1
decorator                4.4.0
deephyper                0.2.1               /home/hyliu/anl/work/deephyper
deepspace                0.0.3               /home/hyliu/anl/work/deepspace
dh-scikit-optimize       0.8.2
filelock                 3.0.12
future                   0.18.2
gast                     0.3.3
google-api-core          1.25.1
google-auth              1.24.0
google-auth-oauthlib     0.4.2
google-pasta             0.2.0
googleapis-common-protos 1.52.0
gpustat                  0.6.0
graphviz                 0.8.4
grpcio                   1.35.0
gym                      0.18.0
h5py                     2.10.0
hiredis                  1.1.0
idna                     2.10
importlib-metadata       3.4.0
Jinja2                   2.11.2
joblib                   1.0.0
jsonschema               3.2.0
Keras                    2.4.3
Keras-Preprocessing      1.1.2
kiwisolver               1.1.0
liac-arff                2.5.0
Markdown                 3.3.3
MarkupSafe               1.1.1
matplotlib               3.1.0
msgpack                  1.0.2
multidict                5.1.0
mxnet                    1.4.1
mypy                     0.710
mypy-extensions          0.4.1
networkx                 2.5
numpy                    1.18.5
nvidia-ml-py3            7.352.0
oauthlib                 3.1.0
opencensus               0.7.12
opencensus-context       0.1.2
openml                   0.10.2
opt-einsum               3.3.0
orderedset               2.0.1
pandas                   0.24.2
Pillow                   7.2.0
pip                      20.3.3
prometheus-client        0.9.0
protobuf                 3.14.0
psutil                   5.6.3
py-spy                   0.3.4
pyaml                    20.4.0
pyasn1                   0.4.8
pyasn1-modules           0.2.8
pydot                    1.4.1
pyglet                   1.5.0
pyparsing                2.4.0
pyrsistent               0.17.3
python-dateutil          2.8.0
pytz                     2019.1
PyYAML                   5.4.1
ray                      1.1.0
redis                    3.5.3
requests                 2.25.1
requests-oauthlib        1.3.0
rsa                      4.7
scikit-learn             0.24.1
scipy                    1.3.0
setuptools               52.0.0.post20210125
six                      1.15.0
tensorboard              2.4.1
tensorboard-plugin-wit   1.8.0
tensorflow               2.3.2
tensorflow-estimator     2.3.0
termcolor                1.1.0
threadpoolctl            2.1.0
tornado                  6.0.2
tqdm                     4.56.0
typed-ast                1.4.0
typeguard                2.10.0
typing-extensions        3.7.4.3
urllib3                  1.26.3
Werkzeug                 1.0.1
wheel                    0.36.2
wrapt                    1.12.1
xgboost                  0.90
xmltodict                0.12.0
yarl                     1.6.3
zipp                     3.4.0
```

`conda list`

```
# packages in environment at /home/hyliu/miniconda/miniconda3/envs/test001:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
absl-py                   0.11.0                   pypi_0    pypi
aiohttp                   3.7.3                    pypi_0    pypi
aiohttp-cors              0.7.0                    pypi_0    pypi
aioredis                  1.3.1                    pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
async-timeout             3.0.1                    pypi_0    pypi
blessings                 1.7                      pypi_0    pypi
ca-certificates           2021.1.19            h06a4308_0  
cachetools                4.2.1                    pypi_0    pypi
certifi                   2020.12.5        py37h06a4308_0  
chardet                   3.0.4                    pypi_0    pypi
click                     7.1.2                    pypi_0    pypi
cloudpickle               1.6.0                    pypi_0    pypi
colorama                  0.4.4                    pypi_0    pypi
colorful                  0.5.4                    pypi_0    pypi
configspace               0.4.16                   pypi_0    pypi
cython                    0.29.21                  pypi_0    pypi
deap                      1.3.1                    pypi_0    pypi
deephyper                 0.2.1                     dev_0    <develop>
deepspace                 0.0.3                     dev_0    <develop>
dh-scikit-optimize        0.8.2                    pypi_0    pypi
filelock                  3.0.12                   pypi_0    pypi
future                    0.18.2                   pypi_0    pypi
gast                      0.3.3                    pypi_0    pypi
google-api-core           1.25.1                   pypi_0    pypi
google-auth               1.24.0                   pypi_0    pypi
google-auth-oauthlib      0.4.2                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
googleapis-common-protos  1.52.0                   pypi_0    pypi
gpustat                   0.6.0                    pypi_0    pypi
grpcio                    1.35.0                   pypi_0    pypi
gym                       0.18.0                   pypi_0    pypi
h5py                      2.10.0                   pypi_0    pypi
hiredis                   1.1.0                    pypi_0    pypi
idna                      2.10                     pypi_0    pypi
importlib-metadata        3.4.0                    pypi_0    pypi
jinja2                    2.11.2                   pypi_0    pypi
joblib                    1.0.0                    pypi_0    pypi
jsonschema                3.2.0                    pypi_0    pypi
keras                     2.4.3                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
ld_impl_linux-64          2.33.1               h53a641e_7  
liac-arff                 2.5.0                    pypi_0    pypi
libedit                   3.1.20191231         h14c3975_1  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 9.1.0                hdf63c60_0  
libstdcxx-ng              9.1.0                hdf63c60_0  
markdown                  3.3.3                    pypi_0    pypi
markupsafe                1.1.1                    pypi_0    pypi
msgpack                   1.0.2                    pypi_0    pypi
multidict                 5.1.0                    pypi_0    pypi
ncurses                   6.2                  he6710b0_1  
networkx                  2.5                      pypi_0    pypi
numpy                     1.18.5                   pypi_0    pypi
nvidia-ml-py3             7.352.0                  pypi_0    pypi
oauthlib                  3.1.0                    pypi_0    pypi
opencensus                0.7.12                   pypi_0    pypi
opencensus-context        0.1.2                    pypi_0    pypi
openml                    0.10.2                   pypi_0    pypi
openssl                   1.1.1i               h27cfd23_0  
opt-einsum                3.3.0                    pypi_0    pypi
pillow                    7.2.0                    pypi_0    pypi
pip                       20.3.3           py37h06a4308_0  
prometheus-client         0.9.0                    pypi_0    pypi
protobuf                  3.14.0                   pypi_0    pypi
py-spy                    0.3.4                    pypi_0    pypi
pyaml                     20.4.0                   pypi_0    pypi
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pydot                     1.4.1                    pypi_0    pypi
pyglet                    1.5.0                    pypi_0    pypi
pyrsistent                0.17.3                   pypi_0    pypi
python                    3.7.9                h7579374_0  
pyyaml                    5.4.1                    pypi_0    pypi
ray                       1.1.0                    pypi_0    pypi
readline                  8.0                  h7b6447c_0  
redis                     3.5.3                    pypi_0    pypi
requests                  2.25.1                   pypi_0    pypi
requests-oauthlib         1.3.0                    pypi_0    pypi
rsa                       4.7                      pypi_0    pypi
scikit-learn              0.24.1                   pypi_0    pypi
setuptools                52.0.0           py37h06a4308_0  
six                       1.15.0                   pypi_0    pypi
sqlite                    3.33.0               h62c20be_0  
tensorboard               2.4.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.0                    pypi_0    pypi
tensorflow                2.3.2                    pypi_0    pypi
tensorflow-estimator      2.3.0                    pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
threadpoolctl             2.1.0                    pypi_0    pypi
tk                        8.6.10               hbc83047_0  
tqdm                      4.56.0                   pypi_0    pypi
typeguard                 2.10.0                   pypi_0    pypi
typing-extensions         3.7.4.3                  pypi_0    pypi
urllib3                   1.26.3                   pypi_0    pypi
werkzeug                  1.0.1                    pypi_0    pypi
wheel                     0.36.2             pyhd3eb1b0_0  
wrapt                     1.12.1                   pypi_0    pypi
xmltodict                 0.12.0                   pypi_0    pypi
xz                        5.2.5                h7b6447c_0  
yarl                      1.6.3                    pypi_0    pypi
zipp                      3.4.0                    pypi_0    pypi
zlib                      1.2.11               h7b6447c_3  
```

**Additional context**

Thank you so much for your help! 

Best regards,
Hongyuan Liu

**Comment by Deathn0t** [comment]
Hello @bigwater, 

Thank you for reporting the issue.

 I will look at the issue today and send you a way to resolve this. In the meantime, [this NAS tutorial](https://deephyper.readthedocs.io/en/latest/tutorials/nas.html) should work better.

**Comment by Deathn0t** [comment]
Hello @bigwater,

The problem should be resolved in 1906703094b6caf5816a587e1fa4ea997b508240
with the two command lines you presented above. 

Let me know if you have issues trying it.

**Comment by bigwater** [comment]
It worked for me. Thank you so much for your help!
